apps/prairielearn/src/lib/file-store.ts

```diff
@@ -191,8 +203,12 @@ export async function getFile(file_id, data_type = 'buffer') {
     }
   }
 
+  const contents = buffer || readStream;
```

Comment:
Nit, this is semantically a little more correct I think.

```suggestion
  const contents = buffer ?? readStream;
```